src/components/shared/svg/email-validation-error.jsx

```diff
@@ -0,0 +1,18 @@
+export default function EmailValidationError() {
```

Comment:
I dont know what this icon looks like, but is there anything that makes it specific to an email? Like there is there an envelope icon? If not, and it is generic, maybe make the name less specific. 
For example `ValidationError` or even just `Error`. And I would do the same for the `EmailValidationSuccess`.